tests/components/unifi/test_sensor.py

```diff
@@ -558,3 +783,27 @@ async def test_wlan_client_sensors(
     mock_unifi_websocket(state=WebsocketState.RUNNING)
     await hass.async_block_till_done()
     assert hass.states.get("sensor.ssid_1").state == "0"
+
+
+async def test_outlet_power_readings(
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker, mock_unifi_websocket
+) -> None:
+    """Test the outlet power reporting on PDU devices."""
+    await setup_unifi_integration(hass, aioclient_mock, devices_response=[PDU_DEVICE_1])
+
+    assert len(hass.states.async_all()) == 5
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 1
+
```

Comment:
```suggestion

    ent_reg = er.async_get(hass)
    ent_reg_entry = ent_reg.async_get("sensor.dummy_usp_pdu_pro_outlet_2_outlet_power")
    assert ent_reg_entry.unique_id == "outlet_power-xxx"
    assert ent_reg_entry.entity_category is EntityCategory.DIAGNOSTIC

```

Reply:
I can't find any information at all about outlet_caps. What I've seen from the API so far:

USP-Plug devices send `"has_metering": false,`
USP-PDU-Pro devices have two different responses:
USB power ports, which have no power stats: `"outlet_caps": 1`
Power outlets, which have power stats: `"outlet_caps": 3,`

My assumption based on the above:
1 == supports switching
[2 == supports metering]
3 == supports switching and metering

I'm wondering if checking if `outlet_current` is defined would be a safer approach? In all three cases above, the outlets that don't have support for providing power stats are not setting this property.

Reply:
I really hope we can use caps like that. Could you detail this in a PR in the library. Did similar for poe just now https://github.com/Kane610/aiounifi/pull/426